File: backend/src/main/java/com/example/backend/requestmodels/AddJewelryRequest.java
package com.example.backend.requestmodels;

import lombok.Data;

@Data
public class AddJewelryRequest {
	
	private String name;
	private String brand;
	private String description;
	private int stockQuantity;
	private double price;        
	private String category;
	private String img;
}

File: frontend/src/models/AddJewelryRequest.ts
class AddJewelryRequest {
    name: string;
    brand: string;
    description: string;
    stockQuantity: number;
    category: string;
    price: number;
    img?: string;

    constructor(
        name: string,
        brand: string,
        description: string,
        stockQuantity: number,
        category: string,
        price: number,
        img?: string
    ) {
        this.name = name;
        this.brand = brand;
        this.description = description;
        this.stockQuantity = stockQuantity;
        this.category = category;
        this.price = price;
        this.img = img;
    }
}

export default AddJewelryRequest;
